modules/registry-change-management-state.adoc

```diff
@@ -14,32 +14,8 @@ To start the image registry, you must change `ManagementState` Image Registry Op
 
 .Procedure
 
-* Change `ManagementState` Image Registry Operator configuration from `Removed` to `Managed`. For example:
+* Change `ManagementState` Image Registry Operator configuration from `Removed` to `Managed`.
 +
-[source,yaml]
 ----
```

Comment:
We've changed our command styling a bit. If you update the block to use this format and squash, I'm happy to merge:
```
[source,terminal]
----
$ oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed"}}'
----
```

Reply:
@kalexand-rh I pulled the latest code and made this change as you requested